src/platforms/mirserver/screen.cpp

```diff
@@ -152,6 +152,11 @@ Screen::Screen(const mir::graphics::DisplayConfigurationOutput &screen)
 {
     setMirDisplayConfiguration(screen, false);
 
+    qCDebug(QTMIR_SENSOR_MESSAGES) << "Screen - identifier is:" << m_orientationSensor->identifier();
+
+    // FIXME: Forcing use of qtubuntu-sensors!
+    m_orientationSensor->setIdentifier("core.orientation");
+
```

Comment:
Is this a case of, the backend is getting connected but the identifier isn't being automatically filled? The lack of a call to `connectToBackend()` anywhere suggests to me that's the case.

If that's not the case would it be better to check the backend connection status first and then explicitly open the connection if need be.

```cpp
if (!m_orientationSensor->connectedToBackend()) {
    m_orientationSensor->setIdentifier(...);
    m_orientationSensor->connectToBackend();
}
```

Reply:
connectToBackend is done by m_orientationSensor->start(); some lines later. 

The backed will never be connect at this stage anyway, since it just initialized it.

Reply:
Excellent! in that case LGTM :+1: